app/views/users/new.html.erb

```diff
@@ -9,6 +9,9 @@
    log in</a>.
    If you don't want to log in with a GitHub account, you can sign up
    here instead.
+   <br><br>
+   If you didn't recieve your activation link, sign up again and we
```

Comment:
`If you didn't receive your activation link, please sign up again and we`

spelling and please